src/components/content/deployServiceConfigParameter/DeploymentServiceConfigurationParameter.tsx

```diff
@@ -0,0 +1,226 @@
+            dataIndex: 'modificationImpact',
+            children: [
+                {
+                    title: <div className={deploymentVariablesStyles.variablesTableChildrenTitle}>Is Data Lost</div>,
+                    dataIndex: ['modificationImpact', 'isDataLost'],
+                    key: 'isDataLost',
+                    render: (text: boolean | undefined | null) => {
+                        if (text === true) {
+                            return <div className={deploymentVariablesStyles.variablesColumns}>{`true`}</div>;
+                        } else if (text === false) {
+                            return <div className={deploymentVariablesStyles.variablesColumns}>{`false`}</div>;
+                        } else {
+                            return null;
+                        }
+                    },
+                },
+                {
+                    title: (
+                        <div className={deploymentVariablesStyles.variablesTableChildrenTitle}>
+                            Is Service Interrupted
+                        </div>
+                    ),
+                    dataIndex: ['modificationImpact', 'isServiceInterrupted'],
+                    key: 'isServiceInterrupted',
+                    render: (text: boolean | undefined | null) => {
+                        if (text === true) {
+                            return <div className={deploymentVariablesStyles.variablesColumns}>{`true`}</div>;
+                        } else if (text === false) {
+                            return <div className={deploymentVariablesStyles.variablesColumns}>{`false`}</div>;
+                        } else {
+                            return null;
+                        }
+                    },
+                },
+            ],
+        },
+        {
+            title: <div className={deploymentVariablesStyles.variablesColumns}>IsReadOnly</div>,
+            dataIndex: 'isReadOnly',
+            render: (text: string) => {
+                return <div className={deploymentVariablesStyles.variablesColumns}>{text}</div>;
+            },
+        },
+        {
+            title: <div className={deploymentVariablesStyles.variablesColumns}>ManagedBy</div>,
+            dataIndex: 'managedBy',
+            render: (text: string) => {
+                return <div className={deploymentVariablesStyles.variablesColumns}>{text}</div>;
+            },
+        },
+    ];
+
+    return (
+        <>
+            <h3 className={catalogStyles.catalogDetailsH3}>
+                <FileTextOutlined />
+                &nbsp;Service Configuration Parameters
+            </h3>
+            <div className={deploymentVariablesStyles.variablesTableContainer}>
+                <Table
+                    columns={columns}
+                    dataSource={parameters}
+                    rowKey={'name'}
+                    bordered
+                    pagination={false}
+                    loading={parameters.length <= 0}
```

Comment:
this loading is not correct.
I see it is wrong in the `DeploymentVariables` component as well. 
It must be based on the API call status and nothing else. 
If there is no need of a loading state, you can just remove it as well.